src/components/Initiatives/InitiativeDetailsPage/InitiativeTableCard/FailingComponentsTable.tsx

```diff
@@ -13,21 +13,45 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import React from 'react';
+import React, { useMemo } from 'react';
+
+import { EmptyState, InfoCard, Table as BSTable, TableColumn } from '@backstage/core-components';
+
 import { InitiativeActionItem } from '../../../../api/types';
-import { Table, TableBody, TableCell, TableRow } from '@material-ui/core';
 import { groupByString } from '../../../../utils/collections';
 import { useDetailCardStyles } from '../../../../styles/styles';
 import { FailingComponentsTableRow } from './FailingComponentsTableRow';
-import { InfoCard } from '@backstage/core-components';
+import { defaultComponentRefContext } from '../../../../utils/ComponentUtils';
+import { humanizeAnyEntityRef } from '../../../../utils/types';
+import { Box } from '@material-ui/core';
+
+const columns: TableColumn[] = [{
+  field: 'all',
+  render: (data: { componentRef?: string, numRules?: number, actionItems?: InitiativeActionItem[] }) => {
+    return (
+      <FailingComponentsTableRow
+        componentRef={data.componentRef ?? ''}
+        actionItems={ data.actionItems ?? [] }
```

Comment:
are the spaces here intentional?

Reply:
No... clearly I blindly rely on a formatter most of the time 😅